src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -370,6 +370,7 @@ Copyright (c) .NET Foundation. All rights reserved.
                               TargetingPackVersion="$(MicrosoftAspNetCoreAppRefPackageVersion)"
                               RuntimePackNamePatterns="Microsoft.AspNetCore.App.Runtime.**RID**"
                               RuntimePackRuntimeIdentifiers="@(AspNetCoreRuntimePackRids, '%3B')"
+                              RuntimePackExcludedRuntimeIdentifiers="android"
```

Comment:
Is it worth adding the other platforms we know about? This pack wouldn't work on `ios` or `maccatalyst` anyway?

Reply:
@jonathanpeppers I think we would only need to do that if `RuntimePackRuntimeIdentifiers` included runtime identifiers that would match `ios` or `maccatalyst`.  The reason we need this change is because the `android` runtime identifiers match the `linux` ones, and there are runtime packs for ASP.NET for linux.